### PR TITLE
 Fixes multiline tags in the public dashboard

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
captura de pantalla 2015-04-16 a las 20 07 51

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3081/))

**Comment by viddo** [comment]
:es:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3084/))

**Comment by alonsogarciapablo** [comment]
@viddo I submitted some more changes. Would you mind taking a look at this again? thx

File: app/assets/stylesheets/new_variables/mixins.css.scss [hunk 1]
```diff
@@ -214,6 +214,6 @@
     right: 0;
     height: $sLineHeight-normal;
     width: 2 * $sLineHeight-normal;
-    @include background-horizontal(rgba(255,255,255,0), rgba(255,255,255,1));
+    @include background-horizontal(rgba(255,255,255,0), inherit);
```
Comment (alonsogarciapablo) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: @viddo what do you think about using `inherit` here?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3085/))

File: app/assets/stylesheets/new_variables/mixins.css.scss (hunk 1 above)
Comment (viddo) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: I didn't implement this mixin and I'm not too confident in this area. From [what I can read up](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient?redirectlocale=en-US&redirectslug=CSS%2Flinear-gradient) the mixin is used for linear gradient, which specifies that the 2nd value is a stop color, so not sure that `inherit` would render consistently across browsers. I'd stick with current definition (i.e. use explicit color value), but @xavijam can probably give a better answer here.

**Comment by viddo** [comment]
:+1:

File: app/assets/stylesheets/new_variables/mixins.css.scss (hunk 1 above)
Comment (alonsogarciapablo) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: Compass is actually spitting the following error:
```
    error tmp/sass/new_common/datasets-list.scss (Line 134 of tmp/sass/new_variables/mixins.scss: Not a valid color stop: Sass::Script::String: inherit)
```

so I'm assuming is not a good idea :(

**Comment by alonsogarciapablo** [comment]
I will release #3263 and refactor it in a separate PR since things were getting complicated.

**Pull request closed without merging**